doc/user/formats.rst

```diff
@@ -124,5 +124,5 @@ fractional_ranges
         Range of the band to mask. The values are between 0.0 and 1.0, with
         0.0 indicating the lowest frequency and 1.0 the highest frequency
         (both in the digitised bandwidth). Channel i should be masked if
-        :math:`\frac{i+0.5}{nchans}` falls into any of the ranges
-        (inclusive).
+        :math:`[\frac{i}{nchans}, \frac{i+1}{nchans}]` overlaps any of the
```

Comment:
@ludwigschwardt I think it should actually be (i-0.5)/nchans to (i+0.5)/nchans, because of the odd way centre frequency is defined. Do you agree?

Reply:
Yes.

Reply:
I guess the way we have it now is that 0.0 is the lowest *channel centre* frequency and 1.0 is the highest channel centre frequency + 1/nchans, but that's not a very nice description.

Reply:
I want to keep the frequencies corresponding to 0.0 and 1.0 independent of the channelisation e.g. L-band is always 856 MHz - 1284 MHz regardless of how many channels there are, and channel 0 extends half a channel below "0.0". See if the new wording is clear for you.

Reply:
Good point. I like the new wording.